woocommerce/woocommerce-android
# Product details instrumentation tests

Fixes #1451. This PR fixes all the failing connected tests that were either out of date or they actually found an issue.

**To test:**
1. Run the `ProductDetailNavigationTest` tests
2. Verify that all of them pass
